src/main/java/org/schemaspy/Config.java

```diff
@@ -285,10 +285,12 @@ public String getTemplateDirectory() {
     }
 
     public boolean isJarFile() {
```

Comment:
This method was created by me to verify if application is execute as `java -jar schemaspy.jar`
I don't remember exactly the issue but it was something related to the layout resources.

Today I tested and everything is working as should :-) I will review the code and try to remove this method.